packages/backend-modules/mail/templates/cf_comment_notification_new.html

```diff
@@ -52,6 +52,19 @@
                       >
                       schreibt:
                     </p>
+                    <p
+                      style="color:#282828;font-size:17px;line-height:24px;{{sg_font_style_sans_serif_regular}}"
+                    >
+                    </p>
+                    <div
+                      style="background-color:#ebf6e5;padding:10px;color:#282828;font-size:17px;line-height:24px;{{sg_font_style_serif_regular}}"
+                    >
+                      {{{COMMENT_TAG}}}
```

Comment:
Triple curly braces [print value raw](https://handlebarsjs.com/guide/expressions.html#html-escaping). We want them to be HTML escaped.